pages/reference/base-images/base-images.md

```diff
@@ -22,10 +22,10 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 	- [Ubuntu](https://www.ubuntu.com/): artful, bionic, cosmic, trusty and xenial
 	- [Fedora](https://getfedora.org/): 26, 28, 29 and 30
 - Multiple language stacks: 
-	- [Node.js](https://nodejs.org/en/): 11.12.0, 10.15.3, 8.15.1 and 6.17.0
-	- [Python](https://www.python.org/): 2.7.15, 3.5.6, 3.6.8 and 3.7.2 
+	- [Node.js](https://nodejs.org/en/): 12.2.0, 11.15.0, 10.15.3, 8.16.0 and 6.17.1
+	- [Python](https://www.python.org/): 2.7.16, 3.5.7, 3.6.8 and 3.7.3 
```

Comment:
Just a minor point, but it looks a bit weird that some language stacks have their versions listed newest -> oldest, and others are listed oldest -> newest.